app/build.gradle

```diff
@@ -213,6 +213,10 @@ dependencies {
     implementation Libs.adjust
     implementation Libs.play_services_analytics // Required by Adjust
 
+    // https://github.com/ReactiveX/RxAndroid
+    implementation 'io.reactivex.rxjava2:rxandroid:2.0.2'
```

Comment:
Could you please help to define the dependency in the file below?
https://github.com/mozilla-tw/Rocket/blob/master/buildSrc/src/main/java/Dependencies.kt

Then you could reference here as:
`implementation Libs.rxandroid`
`implementation Libs.rxjava`